crates/valence/examples/fixed_tick.rs

```diff
@@ -0,0 +1,74 @@
+#![allow(clippy::type_complexity)]
+
+use valence::prelude::*;
+use valence_client::message::SendMessage;
+
+const SPAWN_Y: i32 = 64;
+
+pub fn main() {
+    tracing_subscriber::fmt().init();
+
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_plugin(TickSystem)
+        .add_startup_system(setup)
+        .add_system(init_clients)
+        .add_system(despawn_disconnected_clients)
+        .add_system(run_every_20_ticks.in_schedule(CoreSchedule::FixedUpdate))
+        .add_system(manual_tick_interval)
+        .insert_resource(FixedTick::new(20))
```

Comment:
Does this mean this only allows for one timer?

Reply:
Functionally it works the same as bevy_timer, so yeah, i think so, which is why I added the other example, or want to run it on a seperate scheduler, or maybe even make it some form of dynamic scheduler if thats possible?